content/en/docs/faq/multi-cluster-management/manage-multi-cluster.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Manage a Multi-cluster Environment on KubeSphere"
+keywords: 'Kubernetes, KubeSphere, federation, multicluster, hybrid-cloud'
+description: 'Understand how to manage a multi-cluster environment on KubeSphere.'
+linkTitle: "Manage a Multi-cluster Environment on KubeSphere"
+weight: 16710
+---
+
+KubeSphere provides an easy-to-use multi-cluster feature to help you [build your multi-cluster environment on KubeSphere](../../../multicluster-management/). This guide illustrates how to manage a multi-cluster environment on KubeSphere.
+
+## Prerequisites
+
+- Make sure your Kubernetes clusters are installed with KubeSphere before you use them as your Host Cluster and Member Clusters. 
+
+- Make sure the cluster role is set correctly on your Host Cluster and Member Clusters respectively, and the `jwtSecret` is the same between them.
+
+- It is recommended that your Member Cluster is a clean environment where no resources have been created on it before it is imported to the Host Cluster.
+
+
+## Manage your KubeSphere Multi-cluster Environment
+
+Once you build a multi-cluster environment on KubeSphere, you can manage it through the central control plane from your Host Cluster as you can select a specific cluster when creating resources. You need to avoid creating resources on your Host Cluster to prevent it from overloading. It is not recommended to log in to the KubeSphere web console of your Member Clusters to create resources on them as these resources won't be synchronized to your Host Cluster for management.
```

Comment:
Once you build a multi-cluster environment on KubeSphere, you can manage it through the central control plane from your Host Cluster as you can select a specific cluster when creating resources. You need to avoid creating resources on your Host Cluster to prevent it from overloading.
->
Once you build a multi-cluster environment on KubeSphere, you can manage it through the central control plane from your Host Cluster. When creating resources, you can select a specific cluster while the Host Cluster should be avoided in case of overload.

Reply:
as these resources won't be synchronized to your Host Cluster for management.
->
as some resources (for example, workspaces) won't be synchronized to your Host Cluster for management.